angular/tracker/src/models.js

```diff
@@ -0,0 +1,224 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.CLASS({
+  package: 'ng.tracker.types',
+  name: 'PropertyRefinement',
+  refines: 'foam.core.Property',
+  properties: [
+    {
+      class: 'String',
+      name: 'ngTemplate'
+    }
+  ]
+});
+
+foam.ENUM({
+  package: 'ng.tracker',
+  name: 'IssueStatus',
+
+  properties: [
+    {
+      name: 'label'
+    },
+    {
+      name: 'isOpen',
+      value: false
+    }
+  ],
+
+  values: [
+    {
+      name: 'NEW',
+      values: { label: 'New', isOpen: true }
```

Comment:
The seaprate values:{} field is no longer required for enums.

Just

{
name: 'NEW',
label: 'New',
isOpen: true
}